docs/reference/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,13 @@
+[[restore-from-snapshot]]
+== Restore from snapshot
+
+Elasticsearch is using snapshots to store a copy of your data outside a cluster. You can restore a snapshot to recover
+indices and data streams for which there are no copies of the shards in the cluster. This can happen after a deletion or
```

Comment:
```suggestion
indices and data streams for which there are no copies of the shards in the cluster. This can happen if the data (indices or data streams) was deleted or
```